app/controllers/admin/products_controller.rb

```diff
@@ -3,16 +3,16 @@
 module Admin
   class ProductsController < AdminController
     def new
-      render :new, locals: { product: Product.new }
+      product = Product.new
+      render :new, locals: { product: product, product_presenter: ProductPresenter.new(product) }
```

Comment:
Zastanawiałem się, czy nie inicializować ProductPresentera w widoku formy, ale tak jest chyba najbezpieczniej.

Reply:
widzialem oba podejscia, ale preferuje to. Raczej oba sa ok